etc/templates/allocation.sh.template

```diff
@@ -3,7 +3,16 @@ nodeset=$NODE_SET
 numcpus=$CPUS
 
 export VERY_RNUM=$RANDOM
-export RUBIN_COLLECTOR_HOST="sdfrome001.sdf.slac.stanford.edu, sdfiana012.sdf.slac.stanford.edu"
+
+if [ $COLLECTOR ]; then
+    echo "COLLECTOR is set."
+    export RUBIN_COLLECTOR_HOST=$COLLECTOR
+else
+    echo "COLLECTOR is empty."
+    export RUBIN_COLLECTOR_HOST="sdfrome001.sdf.slac.stanford.edu, sdfiana012.sdf.slac.stanford.edu"
+fi
+echo "Using RUBIN_COLLECTOR_HOST $RUBIN_COLLECTOR_HOST"
+
 export RUBIN_COLLECTOR_PORT=9618
```

Comment:
Since adding the capability to use different host, is it the right time to also allow the specification of the port just in case?

Reply:
My first thought was that it should not be necessary. But I did consider the possibility of running a CM on a Slurm worker node, and then users could run glideins on the same worker node. Will condor daemons instinctively try to use the shared port 9618? Yes possible, so I'll try to add the knob too.